lib/widgets/message_list.dart

```diff
@@ -93,6 +81,13 @@ class _MessageListState extends State<MessageList> {
     final length = model!.messages.length;
     assert(model!.contents.length == length);
     return StickyHeaderListView.builder(
+        // Dismiss compose-box keyboard when dragging up or down.
+        // TODO(?) refine with custom implementation, e.g.:
+        //   - Only dismiss when scrolling up to older messages
+        //   - Show keyboard (focus compose box) on overscroll up to new
+        //     messages (OverscrollNotification)
+        keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
```

Comment:
Hmm. Playing with this behavior, I think I'm happier without it — seems like it'd be annoying when writing something where I want to refer back to things people said earlier.

I just did a quick small survey, and behavior in other messaging apps varies:
* Discord leaves the keyboard up if you scroll (on Android, anyway; the below are all on Android)
* Messages, i.e. the stock SMS app, dismisses it… ah, in fact, dismisses it only if you scroll up, not down
* Signal leaves the keyboard up

Reply:
Sure; removed. It would be good to find a way to let the user dismiss the keyboard, even if this isn't it.